### DISCUSSION TITLE
ROI calibration

### DISCUSSION THREAD

**Discussion by shubhambooking2** [google/meridian#136] [General]
i understand for ROI calibration, we need to pass the distribution parameters for the channel we want to calibrate with CLS, for example lognormal distribution with corresponding mu & sigma. Is there a way to pass directly ROI values from CLS study instead of distribution since based on std dev of distribution the pdf of distribution can vary significantly

**Comment by eddieligo** [comment]
Meridian currently requires passing distributions for ROI calibration; point estimates from lift studies will not suffice. We recommend setting the prior standard deviation informed by the experiment standard deviation. However, if you have additional information or beliefs about the effectiveness of a channel, you can adjust the standard deviation accordingly. Here are two possible cases:

- If you have a strong belief in the effectiveness of a channel, informed by consistent results from multiple lift experiments or other reliable analyses, you can apply an adjustment factor to the standard deviation to reflect your confidence. For example, suppose you have conducted several lift studies for a particular channel, and all the experiments yielded similar ROI point estimates, or you have historical data from previous MMM analyses that support the effectiveness of this channel. In this case, you could use a normal distribution centered at the ROI point estimate and manually set a smaller variance, so that the distribution won't vary too much. This tighter distribution would indicate your strong confidence in the experimental results.

- On the other hand, if you have reservations about the experiment results or believe that market conditions and consumer behavior have changed significantly since the experiment was conducted, you may choose to apply an adjustment factor that reduces your confidence in the experiment results. In this case, you could use a normal distribution centered at the ROI point estimate but with a larger variance than the experiment's standard deviation, depending on your level of skepticism. This wider distribution would reflect your uncertainty about the applicability of the experiment results to the current MMM time window.

It's worth noting that the experimental results may not reflect the modeling window ROI that MMM aims to measure and may not be representative of overall media performance. You could consider using the `roi_calibration_period` argument in `ModelSpec` (see section:  [Set the ROI calibration period](https://developers.google.com/meridian/docs/user-guide/configure-model?hl=en)). This argument provides the flexibility for you to calibrate the MMM using a subset of data by setting the calibration window as the experiment period, or to calibrate the MMM using the complete data, depending on whether the experimental results accurately represent the average media effect within the modeling window.
